config/dr-cluster/manager/ramen_manager_config.yaml

```diff
@@ -13,3 +13,4 @@ ramenControllerType: dr-cluster
 maxConcurrentReconciles: 50
 volSync:
   destinationCopyMethod: Direct
+ramenOpsNamespace: ramen-ops
```

Comment:
This works only for upstream - do we use this config with ocp? is it part of the ramen bundle?

Reply:
Discussed in meeting, this is fine.